package.json

```diff
@@ -64,10 +64,9 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
-    "@react-navigation/native": "5.9.2",
-    "react": "16.13.1",
```

Comment:
this is not needed, because `react` version will be implied by `react-native` itself.